### DISCUSSION TITLE
Auto Respawn on death

### DISCUSSION THREAD

**Discussion by Snele** [Zrips/CMI#5671] [Feature Requests]
Add automatic respawn after death. This will help avoid possible problems with the screen or portal. (It is not necessary to appear on the spawn).

**Comment by dkalaxdk** [comment]
Grant this permission:

![image](https://user-images.githubusercontent.com/13720707/121801927-21abaa80-cc3a-11eb-8bde-533e3f288933.png)

https://www.zrips.net/cmi/permissions/

Should do what you're after.
